conf_example/mosquitto.conf

```diff
@@ -3,14 +3,12 @@ persistence_location /var/lib/mosquitto/
 
 log_type all
 
-listener 1883
-
 #cafile /etc/letsencrypt/live/example.com/chain.pem
 #certfile /etc/letsencrypt/live/example.com/cert.pem
 #keyfile /etc/letsencrypt/live/example.com/privkey.pem
 
-listener 1884
+listener 9001
 protocol websockets
 
 allow_anonymous false
-include_dir /etc/mosquitto/conf.d
\ No newline at end of file
+include_dir /mosquitto/conf/conf.d
```

Comment:
Missing empty line at EOF.

Reply:
Irrelevant now - I removed this commit from the PR